client-java/controller/src/main/java/org/evomaster/client/java/controller/internal/SutController.java

```diff
@@ -685,6 +690,29 @@ public final DbInfoDto getSqlDatabaseSchema() {
             return schemaDto;
         }
 
+        DbInfoDto schema = extractSqlDbSchemaAndConstraints();
```

Comment:
is this `schema` object used? need some clarifying comment, or more descriptive function names

Reply:
this has been changed now.